src/rank_llm/scripts/convert_json_to_jsonl.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import json
+import os
+
+def convert_json_to_jsonl(input_file, output_file):
```

Comment:
why is this needed?

Reply:
Convert from unsupported JSON request format to newer JSONL request format.